.travis.yml

```diff
@@ -16,6 +16,7 @@ env:
   - secure: UYvRRq3/WSAF5udIroJRoN0i4ha04K9tWf5ozXAidP9oF1o3kAdz7dMlb/UWE10KCki8WwYSKufsJknSC7yygg6BgkajpUFWZc36XWUgyIlXEti3kxjq8GhHUtVQpdH/Mp702DgzJCgA4Bwj9TlfkyP+PIYW4PfljRllJtPzqgxSur++23Q+kMmvA8T/GOEyab72ZjEMQmonl0Vxf6UWx/y7/4+XLj34OTzoYQ18utWfH9o9i1KUA8dYFCT3oCauGXF8Ra6iOPmNhBjvKrT9+foyYvfbwtL+o+tcbyBQM+p2toWUC2E5e+gIyed+woNnUMFGjGzkwzRe5evhH7RbssGMnkAHn19EGht+Ycdo5Wqh06kmb6sGVFa5EfvuX8AUHaOJrjBjS4ojHA9rmrkCrLvXpKECJi/NK2if5mk39ULtmtUzBBAjgPY3ZY34wweKrRgKz1Q2+Z59nJuX4/UM/KP3lrG0IWKIqdcrnoRaXsWADmf512zFw699+3rre6Bv7h08cyJZWAPWFhZiFibFOQXpT7uVeGyF2VcNrDjVzOiUFOWb7+5IRLi3Volp9o5OHuH1aOw3vNb2KqbIvJAFcQJF4l7c8Jopp6obeyBtXsupAVOTKZQnc9rokwDvyLbCNfseXXWNPekRD9+Ey8asSM+NauWLASFIWxNrRj6M5m0=
   # NPM_TOKEN
   - secure: "pfovyWN6ILBR8Lpxi3QSWfDuVmDKWC2G+/7hyFGDkY3tjiiAwMNrp/i1NXxcbTzMkPGrzszYIhg2b9VaMUFUamrgYHGuI7YVXBsLPC8nc17Bxx8DHAeGwVf4zxCvgZw7FmTbTt0GURmNfd7TQUUKwATEme/RVGi+c3SLLxE3+6s53if5dzI0uccR6uK2KhvfZ0nXSIBMM5/sQah69vxVD4hVI6jjcRCXxvs5ZpnQqe9LO3mGO4SGmq8Gt4ZPKbKUhDIhNJzS5IUNjXxtS9/Z2SyxfqacYMIScCke/Mw5GBG9mT4bgGSBjFm3/HJjrYqJrZLmcxLqtIdbiM+6X+oTmlocn7l5txayxO4mSqsK6fZwPZNc36seJsWaF5gd6QWOaz4Vqb232Kiqu9h9eTsweO3BAvABQyDVpuNzrkbYGTn3rEM2GKctjtaH7xOoiXXL5eBekeqxbPhpIijRfm5+O30S5ays/e1fDoWp6MuME9oX+yv2Sq1jcdYX83lIWczz+4NViFzRM28U39UYISpM3G8ukmmk02+F4x1tLCXxzgKfPAyWc4YoqexYSEQIX7a88HytIJtZB17Xd8EjzX2YeLZlq0oswIS7tTVfZ/cS2vQ2vTRyqjSWoCl2cv/aJJLRWc8Tih7t+tklm+NlxIy8iOVsKrvvPhAzIFBfDYoTxYI="
+  - GTM_TAG: GTM-53XG2JT
```

Comment:
I think this needs to look more like the environment variables above – e.g. `GTM_TAG="GTM-53XG2JT"`.

Could also do with a comment explaining what it is (`# Google Tag Manager ID – different for production and preview builds` or something like that).